GTG/core/base_store.py

```diff
@@ -136,6 +136,13 @@ def remove(self, item_id: UUID) -> None:
         self.emit('removed', str(item_id))
 
 
+    def batch_remove(self,item_ids: List[UUID]):
+        """Remove multiple items and their descendants at the same time. """
```

Comment:
```suggestion
    def batch_remove(self,item_ids: List[UUID]) -> None:
        """Remove multiple items, ensuring nothing gets deleted twice"""
```
The method actually doesn't do anything about descendants - what it actually does it making sure `remove()` doesn't crash. So I'd like the docstring to convey that if you want to delete several items you need to call this method instead of calling `remove()` in a loop. There is probably a better way to phrase it.